jina/parsers/peapods/base.py

```diff
@@ -69,3 +69,12 @@ def mixin_base_ppr_parser(parser, with_identity: bool = True):
             if _SHOW_ALL_ARGS
             else argparse.SUPPRESS,
         )
+
+    gp.add_argument(
+        '--workspace-id',
+        type=str,
+        default=random_identity(),
+        help='the UUID for identifying the workspace. When not given a random id will be assigned.'
+        'Multiple Pea/Pod/Flow will work under the same workspace if they share the same '
+        '`workspace-id`.',
```

Comment:
```suggestion
        '`workspace-id`.' if _SHOW_ALL_ARGS else argparse.SUPPRESS,
```
don't think user need to know about this, let's hide it from CLI